PTerm-Core/PTerm.class.st

```diff
@@ -20,15 +19,10 @@ PTerm >> announcer [
 
 { #category : #protocol }
 PTerm >> close [
-
 	self master ifNotNil: [
-		usedFallback ifFalse: [
-			pid ifNotNil: [ 
-				self lib kill: pid signal: self lib class SIGHUP ]
-		] ifTrue: [
 			self nextPutAllCr: 'exit'.
 			pid ifNotNil: [ 
-				self lib kill: pid signal: self lib class SIGKILL ] ] ]
+				self lib kill: pid signal: self lib class SIGKILL ] ]
 ]
```

Comment:
Sorry for being a bit late, but I do have one more comment: for `PTerm>>#close`, I would suggest to do what the previous implementation did when ‘usedFallback’ was false (see pull request #23). So:

```smalltalk
close
	self master ifNotNil: [
		pid ifNotNil: [ 
			self lib kill: pid signal: self lib class SIGHUP ] ]
```